build/haumea/zrepl.nix

```diff
@@ -33,23 +33,34 @@
       };
       pruning = {
         keep_sender = [
+          { type = "not_replicated"; }
           {
             type = "grid";
             regex = "^zrepl_snap_.*";
             grid = lib.concatStringsSep " | " [
-              "4x15m"
-              "24x1h"
-              "4x1d"
-              "3x1w"
+              "1x1h"
+              "1x2h"
+              "1x4h"
+              # "grid" acts weird if an interval isn't a whole-number multiple
+              # of the previous one, so we jump from 8h to 24h
```

Comment:
Not sure if it's worth trying to explain the weirdness.  I base it not on actual experience but on definition in their docs &ndash; and how such a model behaves then when running continuously.